js/jquery/Config.hx

```diff
@@ -84,16 +84,21 @@ class Config {
 		// add @:native
 		switch (clsType.name) {
 			case "JQuery":
-				clsType.meta.add(":native", [macro $v{native}], clsType.pos);
+				if(Context.defined('hxnodejs')) clsType.meta.add(":jsRequire", [macro "JQuery"], clsType.pos);
```

Comment:
I think it should be `require("jquery")` instead of `require("JQuery")`, given the npm module is https://www.npmjs.com/package/jquery. Am I right?